### DISCUSSION TITLE
Assets are missing from project 'ner_pytorch_medical'

### DISCUSSION THREAD

**Discussion by Maddocent** [explosion/spaCy#8734] [Help: Coding & Implementations] [answered]
I am running a `spacy` project workflow (git clone explosion/projects) to experiment with spacy projects. Being interested in BioNLP, I wanted to try the project in the repo called 'ner_pytorch_medical'. I am running into issues with the assets of this project. Trying to run the command `python -m spacy project assets` from within the folder containing this project gets me:

```
ℹ Fetching 4 asset(s)
⚠ No URL provided for asset. You need to add this file yourself:
/Users/marcteunis/Documents/workspaces/spacy_projects/tutorials/ner_pytorch_medical/assets/n2c2_2011/i2b2_Beth_Train_Release.tar.gz
⚠ No URL provided for asset. You need to add this file yourself:
/Users/marcteunis/Documents/workspaces/spacy_projects/tutorials/ner_pytorch_medical/assets/n2c2_2011/i2b2_Partners_Train_Release.tar.gz
⚠ No URL provided for asset. You need to add this file yourself:
/Users/marcteunis/Documents/workspaces/spacy_projects/tutorials/ner_pytorch_medical/assets/n2c2_2011/Task_1C.zip
✔ Asset already exists:
```

This looks like an omission in the `project.yml` file supplied in this project.
Any help here would be very much appreciated.

I am running this in VSCode on MacBook (BigSur - M1). Using a virtual env with spacy Version>3 installed and the virtual env activated.

Kind regards

**Comment by BramVanroy** [comment]
Seems to me that they cannot be automatically downloaded because they are behind a [login-wall](https://portal.dbmi.hms.harvard.edu/projects/n2c2-nlp/). Not sure where you got `Task_1C.zip` from though or how it was downloaded because I can't find an URL for it in the YAML either.

**Comment by adrianeboyd** [comment, marked as answer]
See the instructions in the project's README here: https://github.com/explosion/projects/tree/v3/tutorials/ner_pytorch_medical#-data

**Reply by Maddocent** [reply]
Dear @adrianeboyd. Thank you very much for this clear answer. I must have missed this when reading the project documentation.

**Reply by PTMMachineLearning** [reply]
@Maddocent Hello I am on a Facebook Pytorch team reaching out to understand the status and if everything is now working?
